### DISCUSSION TITLE
Coupling terms for optimization problem

### DISCUSSION THREAD

**Discussion by Anonymousnake** [yalmip/YALMIP#1419] [Problems using YALMIP]
Dear Professor, I'm a starter for YALMIP. I've used LMI Toolbox in MATLAB for solving the optimization problem.
I wonder is it manageable to handle the coupling term problem?
Here is the optimization problem
![image](https://github.com/yalmip/YALMIP/assets/38578439/e4d05221-3a15-44de-94a9-3be5f9df524e)
where (20) (1,1)LMI, tau1 and Ne1is coupling.

Besides, if I replace the tau1 in (20) (1,1)LMI to fixed 0.5, it still failed to solve the optimization problem for using the SDPT3 and SEDUMI solver, but it seems feasible for LMILAB solver and MATLAB LMI Toolbox.

tau1 = sdpvar(1);
tau2 = sdpvar(1);
Ye1 = sdpvar(5,1); 
Ne1 = sdpvar(5, 5); 

Z = [0.5 * Ne1, zeros(size(Ne1, 1), 1), (Ne1 * A - Ye1 * C).';
    zeros(1,5), tau2 * P_w, (Ne1*F).';
    Ne1 * A - Ye1 * C,  Ne1*F, Ne1];

Constraints = [Z >= 0, 1 - tau1 - tau2 >= 0, tau1 >= 0, tau2 >= 0, Ne1 >= 0];

Objective = trace(Ne1);

options = sdpsettings('solver', 'sedumi'); 
sol = optimize(Constraints, -Objective, options);

**Comment by johanlofberg** [comment]
yes it is a trivial problem. if it works with one solver it should work in all and vice versa. if not you will have to post a reproducible example

the problem is ill-posed/trivial though as all zeros always is feasible and optimal
